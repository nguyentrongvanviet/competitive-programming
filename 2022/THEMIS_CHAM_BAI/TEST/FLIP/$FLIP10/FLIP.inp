10 4
8 5
1 1
2 7
7 2
9 9
1 7
5 2
7 7
8 8
9 6
8
4
1
2
