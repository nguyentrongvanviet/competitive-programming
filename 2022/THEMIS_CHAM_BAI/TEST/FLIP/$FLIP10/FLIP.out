5 1 7 7 9 7 5 7 8 9 